src/style.css

```
@@ -1,3 +1,83 @@
+/* stylelint-disable import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  background-color: #ccc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  height: 60px;
+  line-height: 60px;
+  box-sizing: border-box;
+  background-color: #ffff;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  bottom: 10px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+}
+
+.nav .nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav .nav__item {
+  margin-right: 20px;
+}
+
+.nav .nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav .nav__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  position: relative;
+  padding: 0;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav .nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav .nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  border-radius: 8px;
 }
```

```suggestion
.is-active::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 0;
  height: 4px;
  width: 100%;
  background-color: #00acdc;
  border-radius: 8px;
}
```
No need to increase selector specificity.